docs/collection-resource.js.html

```diff
@@ -106,9 +106,9 @@ <h1 class="page-title">Source: collection-resource.js</h1>
    * @param {String} id The id of the collection to fetch.
    * @return {Promise|GraphModel} A promise resolving with a `GraphModel` of the collection.
    */
-  fetchWithProducts(id) {
+  fetchWithProducts(id, {productsFirst = 20} = {}) {
```

Comment:
First relevant change, LGTM, although maybe someone should have updated the comment too when this change was made.

Reply:
That's me, I totally forgot to update the comment

Reply:
That's fine! Could you do that when you have a moment and we'll update the documentation again? I don't think that's blocking for this PR.